pha-admin/src/main/java/com/store/pharmacy/Company/serviceImpl/CompanyServiceImpl.java

```diff
@@ -0,0 +1,91 @@
+package com.store.pharmacy.Company.serviceImpl;
```

Comment:
Rename a pkg name. For example:
<img width="202" alt="image" src="https://user-images.githubusercontent.com/86360944/137624290-44ff769d-0148-4ede-b03d-cfa9ffc37b6b.png">